tee-worker/scripts/litentry/migrate_worker.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Copy integritee-service binary and enclave_signed.so to ./tmp/w0
```

Comment:
I think in general it's always good practice to `cd` to the desired working dir first - you never know who is calling the script in which location.

Same applies to other `sh` scripts, you can refer to other existing scripts

Reply:
All small scripts moved into main script `deploy.sh`